modules/external/libScePlayGo/entry.cpp

```diff
@@ -1,130 +1,232 @@
+#include "chunkreader.hpp"
 #include "common.h"
+#include "core/fileManager/fileManager.h"
 #include "logging.h"
 #include "types.h"
 
+#include <chrono>
+#include <mutex>
+
 LOG_DEFINE_MODULE(libScePlayGo);
 
 namespace {
-static uint32_t g_chunks_num = 0;
+static GoReader*             g_goread = nullptr;
```

Comment:
struct pimpl and only one static

Reply:
There's a Handle creating function, I thought put this class generation there later and link handles with readers, but it seems like there's only one handle in every game, so probably using pimpl is more reasonable